### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by akash0x53**

### CODE REVIEW

File: linux-bootstrap-1.md [hunk 1]
```diff
@@ -456,7 +456,7 @@ Jump to main
 That's all, we have stack, bss and now we can jump to `main` C function:
 
 ```assembly
-	calll main
+	call main
```
Comment (0xAX) [inline linux-bootstrap-1.md:459]: It is a not error

File: linux-bootstrap-1.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ Magic power button, what's next?
 Despite that it is series of posts about linux kernel, we will not start from kernel code (at least in this paragraph). Ok, you pressed magic power button on your laptop or desktop computer and it started to work. After this mother board sends signal to the [power supply](http://en.wikipedia.org/wiki/Power_supply) which provides computer with the proper amount of electricity. Once motherboard receives [power good signal](http://en.wikipedia.org/wiki/Power_good_signal), it tries to run CPU. CPU resets all leftover data in its registers and sets up predefined values for every register.
 
 
-[80386](http://en.wikipedia.org/wiki/Intel_80386) and later CPUs defines following predifined data in CPU registers after computer resets:
```
Comment (0xAX) [inline linux-bootstrap-1.md:29]: It is already fixed in master

File: linux-bootstrap-1.md [hunk 3]
```diff
@@ -225,7 +225,7 @@ X+08000  +------------------------+
 
 ```
 
-So after bootloader trasferred control to the kernel, it starts somewhere at:
+So after bootloader transferred control to the kernel, it starts somewhere at:
```
Comment (0xAX) [inline linux-bootstrap-1.md:228]: already in master

**Comment by 0xAX** [comment]
@akash0x53 thank you for PR :+1: 

I left some comments, please rebase on master and push again, thank you.

File: linux-bootstrap-1.md (hunk 1 above)
Comment (akash0x53) [inline linux-bootstrap-1.md:459]: yeah, seen previous PRs. Obvious misleading ain't it?

Comment (0xAX) [inline linux-bootstrap-1.md:459]: > Obvious misleading ain't it?

yeah :)

**Comment by 0xAX** [comment]
@akash0x53 Will you update PR?

**Comment by 0xAX** [comment]
I close this PR to not mix all PRs, please feel free to reopen it if there will be updates

**Pull request closed without merging**